lego/apps/comments/templates/comments/email/comment.html

```diff
@@ -20,13 +20,19 @@
 
     <tr>
         <td class="content-block">
-            {{author_name}} har kommentert på {{ target }}.
+            {{author_name}} har kommentert på {{ target }}:
         </td>
     </tr>
 
     <tr>
         <td class="content-block">
-            Les mer på <a href="{{ frontend_url }}/">{{ frontend_url }}</a>.
+            {{ text|safe }}
+        </td>
+    </tr>
+
+    <tr>
+        <td class="content-block">
+            Les mer på <a href="{{ url }}/">{{ url }}</a>.
```

Comment:
This URL already has a trailing slash from `get_aboslute_url` right? So now it has `//` at the end